runtime/ts/plan/suggestion.ts

```diff
@@ -45,6 +44,24 @@ export class Suggestion {
     this.arc = arc;
   }
 
+  get descriptionText() {
+    return this.getDescription('text') as string;
+  }
+
+  getDescription(modality): string|{} {
```

Comment:
(as with previous CL: might be good to start providing types for these input parameters)